dcgm/pyproject.toml

```
@@ -0,0 +1,62 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+    "setuptools>=66; python_version > '3.0'",
+    "setuptools; python_version < '3.0'",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-dcgm"
+description = "The dcgm-exporter check"
+readme = "README.md"
+license = {text = "BSD-3-Clause"}
```

```suggestion
license = "BSD-3-Clause"
```

Updated in [this PR](https://github.com/DataDog/integrations-core/pull/14744) 😉

---

✅